docs/build-eval.md

```diff
@@ -4,18 +4,6 @@ This document walks through the end-to-end process for building an eval, which i
 
 The steps in this process are building your dataset, registering a new eval with your dataset, and running your eval. Crucially, we assume that you are using an [existing eval template](eval-templates.md) out of the box (if that's not the case, see [this example of building a custom eval](custom-eval.md)). If you are interested in contributing your eval publically, we also include some criteria at the bottom for what we think makes an interesting eval.
 
-We are looking for evals in the following categories:
```

Comment:
Please do not remove this list or maybe it's better in [PULL_REQUEST_TEMPLATE.md](https://github.com/openai/evals/blob/main/.github/PULL_REQUEST_TEMPLATE.md)

Reply:
Oh oops, didn't mean to. Looks like I messed up the versioning.